steam/ext/csgo/client.py

```diff
@@ -58,6 +62,71 @@ async def _handle_ready(self) -> None:
         self.http.user = ClientUser(self._connection, await self.http.get_user(self.user.id64))
         await super()._handle_ready()
 
+    @overload
+    async def inspect_item(self, *, owner: Type[SteamID], asset_id: int, d: int) -> PreviewDataBlock:
```

Comment:
```suggestion
    async def inspect_item(self, *, owner: SteamID, asset_id: int, d: int) -> PreviewDataBlock:
```
This type hint is wrong